db/migrations/0014_add_provider_rsr_scores.sql

```diff
@@ -0,0 +1,25 @@
+-- Migration number: 0014 	 2025-01-27T00:00:00.000Z
+CREATE TABLE IF NOT EXISTS provider_rsr_scores (
+  provider_address TEXT NOT NULL,
+  score REAL NOT NULL,
+  calculated_at DATETIME NOT NULL,
+  calculation_period_start DATETIME NOT NULL,
+  calculation_period_end DATETIME NOT NULL,
```

Comment:
```suggestion
  day DATETIME NOT NULL,
```

Let's simplify this and replace this with a single column `day`. We can then have a row per day.